wedding_site/models/rsvp.py

```diff
@@ -3,13 +3,11 @@
 class RSVP(db.Model): 
     name = db.Column(db.String())
     email = db.Column(db.String(), primary_key=True)
-    number_of_guests = db.Column(db.Integer())
     meal_preference = db.Column(db.String())
 
-    def __init__(self, name, email, number_of_guests, meal_preference): 
+    def __init__(self, name, email, meal_preference): 
```

Comment:
We need to discuss meal pref. Also RSVP option should be invisible until we send invitations.